ai_database_explorer/tasks/run_select_query.airplane.ts

```diff
@@ -0,0 +1,53 @@
+import airplane from "airplane";
+import { Parser, AST } from "node-sql-parser";
+
+export default airplane.task(
+  {
+    slug: "demo_run_select_query",
+    name: "Run select database query",
+    description:
+      "This task runs the provided query against the database resource after verifying it's a SELECT query.",
+    parameters: {
+      query: {
+        type: "sql",
+        name: "Database query",
+        description: "A query you want to run against your database resource",
+        required: true,
+      },
+    },
+    resources: {
+      // Change this to the database resource you want to query
+      db: "demo_db",
+    },
+  },
+  async (params) => {
+    const parser = new Parser();
+
+    let ast: AST[] | AST;
+    try {
+      // We use parser.astify to parse the query and decide whether the given query is a "select" query
+      const parserResult = parser.astify(params.query);
+      if (Array.isArray(parserResult)) {
+        ast = parserResult;
+      } else {
+        ast = [parserResult];
+      }
+    } catch (error) {
+      throw new Error(
+        `Your query could not be parsed. Check your query syntax and run your query again.`
+      );
+      return;
+    }
+
+    for (const query of ast) {
+      if ("select" !== query.type) {
+        throw new Error(
+          `Only SELECT-type queries are allowed. Please provide a different query.`
+        );
```

Comment:
same for above error
```suggestion
          "Only SELECT-type queries are allowed. Please provide a different query."
        );
```